masterlist.yaml

```
@@ -5259,7 +5259,10 @@ plugins:
 
   - name: 'Stm_DiamondCityExpansion.esp'
     url: [ 'https://www.nexusmods.com/fallout4/mods/21853' ]
-    after: [ 'Depravity.esp' ]
+    after:
+      - 'Depravity.esp'
+      - 'DCGuard_Overhaul.esp'
```

```suggestion
      - 'DCGuard_Overhaul.esp'
      - 'Depravity.esp'
```
@such-dev Would you mind sorting the `after` rules alphabetically? It's good to go otherwise.